CONTRIBUTING.md

```diff
@@ -39,33 +42,52 @@ By default, turn off:
 
 But make them configurable, so `--persistence` would be `false` by default for an app like Postgresql, but easy to enable.
 
-### Recommended architectures for binaries in `arkade get`
+### Recommended way for adding binaries to `arkade get`
 
 For installable CLIs, add the following architectures when available:
 
-For Darwin (MacOS):
-
+**For Darwin (MacOS):**
 * AMD64 / `x86_64` (Intel)
-* ARM64 / `aarch64` (M1)
-
-For Linux:
+* ARM64 / `arm64` (M1)
 
+**For Linux:**
 * AMD64 / `x86_64` (Intel/AMD)
 * ARM64 / `aarch64` (64-bit Raspberry Pi OS, AWS Graviton and commercial 64-bit ARM servers)
 * ARMv7 / `armhf` (32-bit Raspberry Pi OS)
 
-Do not add ARMv6 support, or 32-bit Intel.
-
-For Windows:
-
+**For Windows:**
 * AMD64 (Intel/AMD)
 
-Do not add ARM support or 32-bit Intel.
+Do not add ARMv6 support, or 32-bit architecture.
 
 FreeBSD and other operating systems are not supported at this time.
 
 Binaries that are packaged as `.zip`, `.tgz` or `.tar.gz` are also supported, look at previous apps to find an example to copy.
 
+Always write a unit tests for your tool in the `pkg/get/get_test.go`.
+
+In the unit tests always pin an exact version and write a test case for that version.
+
+You can try different architectures and OSes with `arkade get` via `--arch` and `--os` flags, to try out arm64 binaries for example.
+
+Please always test your tools manually (`hack/test-tool.sh`) and with the end-to-end test tool (`make e2e`).
```

Comment:
Would it be useful if we also explained what and what Version / VersionNumber are, and showed some sample values for these fields?

Reply:
I think line [86-87](https://github.com/alexellis/arkade/pull/741/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R86-R87) covers this just fine, but it would be a nice idea to provide example values there for sure!
Should I add some?

Reply:
I'd like to see an example set of values, what someone can expect, format it as a JSON object if you like?

Reply:
@Shikachuu when you get to this, message me on discord so I can merge the PR.